dashboards/victoriametrics.json

```diff
@@ -1015,7 +1015,7 @@
       "datasource": {
         "uid": "$ds"
       },
-      "description": "VictoriaMetrics stores various caches in RAM. Memory size for these caches may be limited with -`memory.allowedPercent` flag. Line `max allowed` shows max allowed memory size for cache.",
+      "description": "* `*` - unsupported query path\n* `/write` - insert into VM\n* `/metrics` - query VM system metrics\n* `/query` - query instant values\n* `/query_range` - query over a range of time\n* `/series` - match a certain label set\n* `/label/{}/values` - query a list of label values (variables mostly)",
```

Comment:
This line looks weird

Reply:
Looks like lines just got reshuflled since old panel was deleted. If you check below, the id on L:1034 has changed, so github unified diff view just comparing wrong lines.
I think it is ok:

<img width="740" alt="image" src="https://user-images.githubusercontent.com/2902918/154476347-908451d4-d4dc-4785-aa49-a438e860cbb0.png">
<img width="966" alt="image" src="https://user-images.githubusercontent.com/2902918/154476372-6f01e220-311e-413f-9c6b-5f9abd65cc75.png">